autogluon/task/tabular_prediction/predictor.py

```diff
@@ -624,6 +627,77 @@ def info(self):
         """
         return self._learner.get_info(include_model_info=True)
 
+    # TODO: Handle cases where name is same as a previously fit model, currently overwrites old model.
+    # TODO: Add dataset argument
+    # TODO: Add option to disable OOF generation of newly fitted models
+    # TODO: Move code logic to learner/trainer
+    # TODO: Add task.fit arg to perform this automatically at end of training
+    def fit_weighted_ensemble(self, base_models: list = None, name_suffix='_custom', expand_pareto_frontier=False, time_limits=None):
+        """
+        [EXPERIMENTAL] Fits new weighted ensemble models.
```

Comment:
"Fits new weighted ensemble models" 
-> 
"Fits new weighted ensemble models to combine predictions of previously-trained models"

Reply:
Added